utils/auth.py

```diff
@@ -0,0 +1,9 @@
+import bcrypt
+
+def hash_pw(plaintext_password):
+    if isinstance(plaintext_password, str):
+        return bcrypt.hashpw(bytes(plaintext_password, 'utf-8'), bcrypt.gensalt())
+
+def check_pw(plaintext_password, db_password):
+    if isinstance(plaintext_password, str):
```

Comment:
Is this type check necessary? What should we do if it is not a string?

Reply:
Nope! I added it initially as a safeguard in case we had the case of "1234"(str) vs 1234(int). I tested the latter, and it still parsed as a string - so I removed the check~